### PR TITLE
Unify build flags function for all tasks

### PR DESCRIPTION
**Pull request by sverdlov93**
- [ ] All [tests](https://github.com/jfrog/jfrog-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] This pull request is on the dev branch.
- [ ] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: jfrog-tasks-utils/utils.d.ts [hunk 1]
```diff
@@ -20,6 +20,9 @@ declare module '@jfrog/tasks-utils' {
     export function handleSpecFile(cliCommand: string, specPath: string): string;
     export function addBoolParam(cliCommand: string, inputParam: string, cliParam: string): string;
     export function addStringParam(cliCommand: string, inputParam: string, cliParam: string, require: boolean): string;
+
+    export function addIntParam(cliCommand: string, inputParam: string, cliParam: string): string;
+
```
Comment (yahavi) [inline jfrog-tasks-utils/utils.d.ts]: Let's be consistent with the no-space convention of this file
```suggestion
    export function addIntParam(cliCommand: string, inputParam: string, cliParam: string): string;
```

File: tasks/JFrogDocker/task.json [hunk 2]
```diff
@@ -125,7 +125,7 @@
             "label": "Allow fail build",
             "defaultValue": "true",
             "required": true,
-            "helpMarkDown": "Unset if you don't wish the audit process to exit with code 3, if a violation matching an Xray policy is found, and the policy includes a 'Fail Build' rule.",
+            "helpMarkDown": "Unset if you don't wish the Scan process to exit with code 3, if a violation matching an Xray policy is found, and the policy includes a 'Fail Build' rule.",
```
Comment (yahavi) [inline tasks/JFrogDocker/task.json]:
```suggestion
            "helpMarkDown": "Unset if you don't wish the scan process to exit with code 3, if a violation matching an Xray policy is found, and the policy includes a 'Fail Build' rule.",
```

**Review by yahavi — CHANGES_REQUESTED** [review]
Please run `npm run format` before the merge

File: tests/tests.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ let tasksOutput: string;
 
 describe('JFrog Artifactory Extension Tests', (): void => {
     let repoKeys: any;
-    before(function(): void {
+    before(function (): void {
```
Comment (sverdlov93) [inline tests/tests.ts]:
```suggestion
    before(function(): void {
```

**Pull request merged**